expected/wasm32-wasi/undefined-symbols.txt

```diff
@@ -19,48 +19,50 @@ __subtf3
 __trunctfdf2
 __trunctfsf2
 __unordtf2
-__wasi_args_get
-__wasi_args_sizes_get
-__wasi_clock_res_get
-__wasi_clock_time_get
-__wasi_environ_get
-__wasi_environ_sizes_get
-__wasi_fd_advise
-__wasi_fd_allocate
-__wasi_fd_close
-__wasi_fd_datasync
-__wasi_fd_fdstat_get
-__wasi_fd_fdstat_set_flags
-__wasi_fd_filestat_get
-__wasi_fd_filestat_set_size
-__wasi_fd_filestat_set_times
-__wasi_fd_pread
-__wasi_fd_prestat_dir_name
-__wasi_fd_prestat_get
-__wasi_fd_pwrite
-__wasi_fd_read
-__wasi_fd_readdir
-__wasi_fd_renumber
-__wasi_fd_seek
-__wasi_fd_sync
-__wasi_fd_tell
-__wasi_fd_write
-__wasi_path_create_directory
-__wasi_path_filestat_get
-__wasi_path_filestat_set_times
-__wasi_path_link
-__wasi_path_open
-__wasi_path_readlink
-__wasi_path_remove_directory
-__wasi_path_rename
-__wasi_path_symlink
-__wasi_path_unlink_file
-__wasi_poll_oneoff
-__wasi_proc_exit
-__wasi_random_get
-__wasi_sched_yield
-__wasi_sock_recv
-__wasi_sock_send
-__wasi_sock_shutdown
+__wasi_real_args_get
+__wasi_real_args_sizes_get
+__wasi_real_clock_res_get
+__wasi_real_clock_time_get
+__wasi_real_environ_get
+__wasi_real_environ_sizes_get
+__wasi_real_fd_advise
+__wasi_real_fd_allocate
+__wasi_real_fd_close
+__wasi_real_fd_datasync
+__wasi_real_fd_fdstat_get
+__wasi_real_fd_fdstat_set_flags
+__wasi_real_fd_fdstat_set_rights
+__wasi_real_fd_filestat_get
+__wasi_real_fd_filestat_set_size
+__wasi_real_fd_filestat_set_times
+__wasi_real_fd_pread
+__wasi_real_fd_prestat_dir_name
+__wasi_real_fd_prestat_get
+__wasi_real_fd_pwrite
+__wasi_real_fd_read
+__wasi_real_fd_readdir
+__wasi_real_fd_renumber
+__wasi_real_fd_seek
+__wasi_real_fd_sync
+__wasi_real_fd_tell
+__wasi_real_fd_write
+__wasi_real_path_create_directory
+__wasi_real_path_filestat_get
+__wasi_real_path_filestat_set_times
+__wasi_real_path_link
+__wasi_real_path_open
+__wasi_real_path_readlink
+__wasi_real_path_remove_directory
+__wasi_real_path_rename
+__wasi_real_path_symlink
+__wasi_real_path_unlink_file
+__wasi_real_poll_oneoff
+__wasi_real_proc_exit
+__wasi_real_proc_raise
+__wasi_real_random_get
+__wasi_real_sched_yield
+__wasi_real_sock_recv
+__wasi_real_sock_send
+__wasi_real_sock_shutdown
```

Comment:
Why is this list two items longer?   (github diff isn't telling me which two are the new ones).

Reply:
The two new items are proc_raise and set_rights. I believe the reason this is showing up now is that wasi-libc never called either of those functions internally, so they were just defined in the header. By having a shim though for all these functions they're all consumed now so they're all showing up as undefined symbols.